extensions/common/src/main/java/com/twocoders/extensions/common/ContextExtensions.kt

```diff
@@ -234,7 +234,12 @@ fun Context.isNightModeEnabled() = when (resources.configuration.uiMode.and(Conf
 }
 
 val Context.displayRotation: Int
-    get() = display?.rotation ?: Surface.ROTATION_0
+    get() = if (isAtLeastR()) {
+        display?.rotation ?: Surface.ROTATION_0
+    } else {
+        @Suppress("DEPRECATION")
+        windowManager?.defaultDisplay?.rotation ?: Surface.ROTATION_0
+    }
```

Comment:
toto mozes zapisat aj tak, aby sa ti neopakovalo `Surface.ROTATION_0`
```suggestion
    @Suppress("DEPRECATION") 
    get() = (if (isAtLeastR()) display?.rotation else windowManager?.defaultDisplay?.rotation) ?: Surface.ROTATION_0
```

s tym formatovanim sa mozes pohrat podla toho ake je to dlhe na 1 riadok atd...